scripts/pylib/twister/twisterlib/environment.py

```diff
@@ -196,6 +196,11 @@ def add_parse_arguments(parser = None):
         help="""Only run device tests with current artifacts, do not build
              the code""")
 
+    test_or_build.add_argument("--timeout-multiplier", type=float, default=1,
```

Comment:
why is this being added to test_or_build? 
What if I want to use this argument with --test-only?, right now I am getting:

```twister: error: argument --test-only: not allowed with argument --timeout-multiplier```

Reply:
Yep, indeed it's not correct. Was confused by a bit misleading `test_or_build name`. :)

Now fixed.